docs/docs/networks/picasso/pica-use-cases.md

```diff
@@ -14,36 +24,63 @@ Apollo is a permissionless, MEV-resistant oracle solution. Anyone can run an ora
-
-For example any cross-chain XCMP transactions can choose which assets to use to pay for the transaction. These tokens are then swapped for PICA under the hood, allowing users and liquidity to flow seamlessly through Picasso regardless of what ecosystem they are arriving from.
+The PICA token is uniquely positioned as the gas token at the center of Picasso, powering Composable’s efforts to enhance blockchain interoperability. PICA will also act as the gas token for the CosmWasm dApps deployed on the `ccw-vm` on Picasso. Notably, in order to further support users from other ecosystems Picasso offers a feature called “bring your own gas” (BYOG), which allows users to pay their gas fee in any supported tokens.
 
 All fees may change dynamically depending on network load and pool or protocol fee formulas. The most fundamental factor for gas fees is the computational resources it consumes which is represented as the transaction's "weight". The weight of a transaction is converted into an appropriate amount of PICA by the polynomial formula which changes dynamically depending on the target load of the network. This means as the usage of the chain increases towards maximum capacity, the price of a unit of weight increases as well.
 
 ### Primary pairing on Pablo​
 
-Pablo is the native DEX of the Picasso ecosystem and is integrated directly into the runtime of our parachain as a pallet. As such, a primary trading pair on Pablo will be PICA. You can also expect various liquidity incentives with 15% of PICA’s supply being allocated towards liquidity programs on Picasso. 
+Pablo is the native DEX of the Picasso ecosystem and is integrated directly into the runtime of our parachain as a pallet. As such, a primary trading pair on Pablo will be PICA. You can also expect various liquidity incentives with 15% of PICA’s supply being allocated towards liquidity programs. 
 
-### Powering our advancement into the Cosmos
-New use cases will continue to be established as we move forward with XCM channel openings and our expansion into the Cosmos.
+### Powering advancements in the Cosmos
+New use cases will continue to be established as we move forward with HRMP channel openings and expansion into and beyond the Cosmos.
 
-## Governance & treasury management
+## Governance
 
-The treasury will be community owned and controlled by PICA token holders.
+Picasso is waging war on centralization with a vision of a seamlessly interoperable, trustless future for DeFi. As such the PICA token will play an important role in helping to realize this vision, as governance will soon be powered by phase 2 of Picasso's [OpenGov](./governance.md).
 
```

Comment:
and governance on cosmos chain , please see the docs i edited and left comments for in the rebrand channel